### PR TITLE
Add more details for error messages

### PR DESCRIPTION
**Pull request by hckhanh**
I think that it's necessary to add more details about invalid paths or methods. I have to modify the node_modules code for myself to debug.

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by ronag** [comment]
CI fails

File: lib/core/request.js [hunk 1]
```diff
@@ -76,13 +76,13 @@ class Request {
     ) {
       throw new InvalidArgumentError('path must be an absolute URL or start with a slash')
     } else if (invalidPathRegex.exec(path) !== null) {
-      throw new InvalidArgumentError('invalid request path')
+      throw new InvalidArgumentError('invalid request path: ' + path)
```
Comment (kibertoad) [inline lib/core/request.js:79]: why not use template string?

**Comment by hckhanh** [comment]
1y pr, but thanks

**Comment by ronag** [comment]
CI is still failing and there has not been any interest in fixing.

**Pull request closed without merging**